src/LaraCart.php

```diff
@@ -465,7 +466,7 @@ public function taxTotal($format = true)
      *
      * @return string
      */
-    public function total($format = true, $withDiscount = true)
+    public function total($format = true, $withDiscount = true, $withTax = true)
```

Comment:
ah! yes forgot you can turn off the discounts. You can keep this now.  Approved :+1: